docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 
 services:
   secret-manager:
-    image: node:12.16-alpine3.10
+    image: node:16-alpine3.15
```

Comment:
make sure we use the same docker image for the Dockerfile https://github.com/teracyhq-incubator/secret-manager-action/blob/develop/Dockerfile#L2